mappings/net/minecraft/client/render/model/BakedModelManager.mapping

```diff
@@ -1,11 +1,17 @@
 CLASS net/minecraft/class_1092 net/minecraft/client/render/model/BakedModelManager
+	FIELD field_20277 colorationManager Lnet/minecraft/class_324;
+	FIELD field_20278 stateLookup Lit/unimi/dsi/fastutil/objects/Object2IntMap;
 	FIELD field_5407 missingModel Lnet/minecraft/class_1087;
-	FIELD field_5408 modelMap Ljava/util/Map;
+	FIELD field_5408 models Ljava/util/Map;
 	FIELD field_5409 spriteAtlas Lnet/minecraft/class_1059;
-	FIELD field_5410 blockStateMaps Lnet/minecraft/class_773;
+	FIELD field_5410 cache Lnet/minecraft/class_773;
```

Comment:
I think that's insufficiently descriptive

Reply:
Perhaps you're right. I double-checked and this field isn't a map as I'd remembered. It's a `BlockModels`.

The name comes from here:

```
    protected void apply(ModelLoader loader, ResourceManager manager, Profiler profiler) {
        profiler.startTick();
        profiler.push("upload");
        loader.upload(profiler);
        this.models = loader.getBakedModelMap();
        this.stateLookup = loader.getStateLookup();
        this.missingModel = this.models.get(ModelLoader.MISSING);
        profiler.swap("cache");
        this.cache.reload();
        profiler.pop();
        profiler.endTick();
    }
```

Though `cache` here could very well be referring to the method `reload()` as that proceeds to repopulate the internal `models` map, which could very well be named `cache` (as it is a cache of computed models).

Or the class itself could be named `BakedModelsCache` (as it only works with/contains `BakedModels` and serves as a buffer in front of the `BakedModelManager`/Factory).

It's less work just to name the field `blockModels` after the class, I suppose. :/